src/magic_folder/web.py

```diff
@@ -114,11 +134,158 @@ class APIv1(Resource, object):
     """
     _global_config = attr.ib()
     _global_service = attr.ib()
+    _tahoe_client = attr.ib()
 
     def __attrs_post_init__(self):
         Resource.__init__(self)
         self.putChild(b"magic-folder", MagicFolderAPIv1(self._global_config))
         self.putChild(b"snapshot", SnapshotAPIv1(self._global_config, self._global_service))
+        self.putChild(b"participants", ParticipantsAPIv1(self._global_config, self._tahoe_client))
+
+
+@attr.s
+class ParticipantsAPIv1(Resource, object):
+    """
+    Implements the ``/v1/participants`` portion of the HTTP API
+    resource hierarchy.
+
+    :ivar GlobalConfigDatabase _global_config: The global configuration for
+        this Magic Folder service.
+    """
+    _global_config = attr.ib()
+    _tahoe_client = attr.ib()
+
+    def __attrs_post_init__(self):
+        Resource.__init__(self)
+
+    def getChild(self, name, request):
+        name_u = name.decode("utf-8")
+        try:
+            folder_config = self._global_config.get_magic_folder(name_u)
+        except ValueError:
+            return NoResource(b"{}")
```

Comment:
I suspect we'll want something smarter than this, but we can file an issue and handle it in a follow up.

Reply:
Not sure what you mean by "smarter"? Do you mean a 404 on a non-existent magic-folder name is insufficient?

Reply:
I'm thinking:
- if the resource we return in the successful case isn't a leaf, will this handle requests to the subresource requests as expected
- do we want richer JSON error messages, particularly in the case with subresources.